python/ray/air/_internal/usage.py

```diff
@@ -18,16 +20,97 @@
     "XGBoostTrainer",
 }
 
+# searchers implemented by Ray Tune.
+TUNE_SEARCHERS = {
+    "AxSearch",
+    "BayesOptSearch",
+    "TuneBOHB",
+    "DragonflySearch",
+    "HEBOSearch",
+    "HyperOptSearch",
+    "NevergradSearch",
+    "OptunaSearch",
+    "SkOptSearch",
+    "ZOOptSearch",
+}
+
+# These are just wrappers around real searchers.
+# We don't want to double tag in this case, otherwise, the real tag
+# will be overwritten.
+TUNE_SEARCHER_WRAPPER = {
```

Comment:
nit: Rename with plural (but also update references)
```suggestion
TUNE_SEARCHER_WRAPPERS = {
```